### ISSUE TITLE
qgis_map.fcgi on macOS doesn't find libraries

### ISSUE THREAD

**Issue by qgib** [qgis/QGIS#29399]
Author Name: **Arno Lepisk** (Arno Lepisk)
Original Redmine Issue: [21583](https://issues.qgis.org/issues/21583)
Affected QGIS version: 3.6.0
Redmine category:qgis_server

---
I try to run the qgis server via spawn-fcgi on macOS. The server seems to start, but when trying to access the webpage I get an `ServerException "Layer(s) not valid"`
The server log says:
```
@11:26:03 WARNING No Data Providers[53228]: No QGIS data provider plugins found in:
/Applications/QGIS3.6.app/Contents/MacOS/fcgi-bin/../PlugIns/qgis
No vector layers can be loaded. Check your QGIS installation
11:26:04 WARNING No Authentication Methods[53228]: No QGIS auth method plugins found in:
/Applications/QGIS3.6.app/Contents/MacOS/fcgi-bin/../PlugIns/qgis
No authentication methods can be used. Check your QGIS installation
11:26:04 WARNING [53228]: Couldn't load Python support library: Cannot load library /Applications/QGIS3.6.app/Contents/MacOS/fcgi-bin/lib/qgispython: (dlopen(/Applications/QGIS3.6.app/Contents/MacOS/fcgi-bin/lib/libqgispython.dylib, 10): image not found)
@
```

It looks like the application is using `@/Applications/QGIS3.6.app/Contents/MacOS/fcgi-bin@` as prefix, when it should use 

```
@/Applications/QGIS3.6.app/Contents/MacOS@ - i.e. the library it tries to load 
(@/Applications/QGIS3.6.app/Contents/MacOS/fcgi-bin/lib/libqgispython.dylib@) is in
 @/Applications/QGIS3.6.app/Contents/MacOS/lib/libqgispython.dylib@
```

**Comment by PeterPetrik** [comment]
Can you please test with https://qgis.org/downloads/macos/nightly2/qgis_nightly2_master_20200923_062253.dmg
You can run `/Applications/QGIS.app/Contents/MacOS/bin/qgis_mapserver` with QGIS_SERVER_LANDING_PAGE_PROJECTS_DIRECTORIES environment variable to your project folder

**Comment by PeterPetrik** [comment]
cc @gioman

**Comment by PeterPetrik** [comment]
I believe this was fixed in QGIS 3.16 releases, @gioman why the Feedback label was removed? Is it not working anymore?

**Comment by gioman** [comment]
>  @gioman why the Feedback label was removed? Is it not working anymore?

@PeterPetrik no, it was probably by mistake.
